docs/UserGuide.adoc

```diff
@@ -329,43 +337,37 @@ List all archived guests
 List all checked-in guests
 // end::list[]
 
+// tag::login[]
 === Login : `login`
 
 Logs in to the Concierge application. +
 Format: `login user/USERNAME pw/PASSWORD`
 
-****
-* The username and password are both case-sensitive.
-* A login allows the user to access the commands which can affect the bookings.
-****
+Note: The username and password are both case-sensitive.
 
-Commands which require login:
+[TIP]
+A login allows the user to access the commands which can affect the
+bookings.
 
 ****
-* `login user/<name> pw/<name>-pw`, where <name> is any of the developer team’s GitHub username
-* As of v1.3, you must create your own passwords.json file with your username and passwords.
-For demo purposes, go to https://github.com/CS2103-AY1819S1-F11-2/main and copy the passwords.json file there
-into the same folder as the Concierge.jar file.
-Alternatively, create a new passwords.json and use SHA-256 Hash to generate
-the hash for your password. You can use this website for this: https://passwordsgenerator.net/sha256-hash-generator/
-
+* The default account can be accessed with `login user/admin pw/passw0rd`
+* More users can be added at `src/data/passwords.json`. When adding passwords
```

Comment:
have you tested with `jar` file?

Reply:
Hmm I tested with `gradlew clean shadowJar`, then copied the jar in `build/jar` to another folder, so it should work.